skopt/parameter.py

```diff
@@ -151,6 +178,8 @@ def __init__(self, low, high, prior='uniform', transformer='identity'):
 
         if transformer == 'identity':
             self.transformer = Identity()
+        elif transformer == 'log':
+            self.transformer = Log()
```

Comment:
Does it make sense to allow log transform for integers? Not sure we should support it, or at least the inverse transform should be cast back to integers.